media-sound/qpwgraph/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>proaudio@gentoo.org</email>
+		<name>Gentoo ProAudio Project</name>
+	</maintainer>
+	<use>
+		<flag name="systray">Enables the system tray</flag>
+	</use>
+	<upstream>
```

Comment:
Since you provide upstream information, please add a `<remote-id>` if possible. I don't know if there is a remote-id for third party GitLab instances, though. If not, just tell me and we can leave it out for now.